.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   # Run all tests before db_block_cache_test (db_test, db_test2)
   - JOB_NAME=unittests ROCKSDBTESTS_END=db_block_cache_test
   # Run all tests starting from db_block_cache_test (db_block_cache_test, ..., plain_table_db_test)
-  - JOB_NAME=unittests ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
+  - JOB_NAME=unittests MEM_ENV=true ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
```

Comment:
If it covers file system calls that are not sufficiently covered by other tests, I suggest we do it in real file system. In this specific case, RandomRWFile's implementation on MAC is not sufficiently covered in our internal tests. We rely on Travis to catch issues.

Are there tests in external_sste_file_test running very slow?